app/main.py

```diff
@@ -1 +1,61 @@
-#  write your code here
+class Animal:
+    animals_list = []
+
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.animals_list.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = not self.is_hungry
+            return self.appetite
+        else:
+            return 0
+
+
+class Cat(Animal):
+
+    def __init__(self,
+                 name: str,
+                 appetite: int = 3,
+                 is_hungry: bool = True
+                 ) -> None:
+        super().__init__(name, is_hungry)
```

Comment:
The `super().__init__` call is missing the `appetite` parameter. It should be `super().__init__(name, appetite, is_hungry)` to properly initialize the `Animal` class.